### PR TITLE
migrate and seed

### PR DESCRIPTION
**Pull request by TueAnhAnh**
ready

### CODE REVIEW

**Comment by TueAnhAnh** [comment]
ready

File: app/Task.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $fillable = ['name'];
```
Comment (quoctd-0607) [inline app/Task.php:9]: protected $fillable = [
'name',
];

File: app/Task.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    public function user()
```
Comment (quoctd-0607) [inline app/Task.php:10]: cách trên 1 line

**Comment by TueAnhAnh** [comment]
ready

**Comment by TueAnhAnh** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**